core/CMakeLists.txt

```diff
@@ -1209,6 +1209,7 @@ DR_export_header(${CMAKE_CURRENT_SOURCE_DIR}/ir/instrlist_api.h dr_ir_instrlist.
 DR_export_header(${CMAKE_CURRENT_SOURCE_DIR}/ir/dr_ir_opcodes.h dr_ir_opcodes.h)
 DR_export_header(${CMAKE_CURRENT_SOURCE_DIR}/ir/x86/opcode_api.h dr_ir_opcodes_x86.h)
 DR_export_header(${CMAKE_CURRENT_SOURCE_DIR}/ir/arm/opcode_api.h dr_ir_opcodes_arm.h)
+DR_export_header(${CMAKE_CURRENT_SOURCE_DIR}/ir/category_api.h dr_ir_categories.h)
```

Comment:
Looks like this exported file will not be included by dr_api.h, unlike all the other files: that doesn't seem like what we want to suddenly require explicit includes for just this one file.

Reply:
Maybe we could remove this header at all? add categories to instr_api.h like dr_fp_type_t structure

Reply:
Yes that sounds fine to me.